File: language-support/js/daml-ledger/src/model/submit_request.ts
// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
// SPDX-License-Identifier: Apache-2.0

import * as ledger from '..';

/**
 * @interface ledger.SubmitRequest
 */
export interface SubmitRequest {
    /**
     * @member {ledger.Commands} commands
     * @memberof ledger.SubmitRequest
     * @instance
     */
    commands: ledger.Commands
}

File: ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/messages/command/submission/SubmitRequest.scala
// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
// SPDX-License-Identifier: Apache-2.0

package com.daml.ledger.api.messages.command.submission

import brave.propagation.TraceContext
import com.daml.ledger.api.domain

case class SubmitRequest(commands: domain.Commands, traceContext: Option[TraceContext])
